### ISSUE TITLE
Vercel deployment should be modified to not require Leap

### ISSUE THREAD

**Issue by johnsiano** [astriaai/headshots-starter#118]
Especially if people are using this on Vercel just to try out the auth with Supabase, they shouldn't be required to enter the Leap webhook secret key (it's not even clear where one would get this).

**Comment by burgalon** [comment]
where are you seeing this? can you share a link + screenshot?

**Comment by D2OKAY** [comment]
This happens when we try to deploy this directly through vercel 
<img width="1370" alt="Screenshot 2024-09-27 at 12 10 59 PM" src="https://github.com/user-attachments/assets/11644e86-f931-4d0d-a587-efed0c902750">

**Comment by burgalon** [comment]
@Precious-Macaulay any idea why this might happen?

**Comment by Precious-Macaulay** [comment]
> @Precious-Macaulay any idea why this might happen?

it is already updated with what i can see here

![image](https://github.com/user-attachments/assets/46fd145b-a8bf-4369-b884-78ded90d67de)

**Comment by burgalon** [comment]
I think the deploy link on the README includes some old references to leap, but I'm not sure how to create a proper deployment link with the updated links and not including LEAP_API_KEY

**Comment by D2OKAY** [comment]
> > This happens when we try to deploy this directly through vercel <img alt="Screenshot 2024-09-27 at 12 10 59 PM" width="1370" src="https://private-user-images.githubusercontent.com/165281611/371378387-11644e86-f931-4d0d-a587-efed0c902750.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc0MzY0MzIsIm5iZiI6MTcyNzQzNjEzMiwicGF0aCI6Ii8xNjUyODE2MTEvMzcxMzc4Mzg3LTExNjQ0ZTg2LWY5MzEtNGQwZC1hNTg3LWVmZWQwYzkwMjc1MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyN1QxMTIyMTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zOWE2ZGMwMWYyNmVmZDczZjYwOWI5ZGQ2NWI3ZGE4MDVmOWZkZGJjMTJiNDAzZTJjYTZiNThmYmQwNzU3NTY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.dfiGj9OMdJ4h0GCV08nmfMk-X9hmaEMWkFLFfV43GMM">
> 
> can you send the vercel deployment like you are using?
> 
> The deploy link in the Readme file is properly configured.

this is the link at vercel deployment : https://vercel.com/templates/next.js/headshot-ai 
- when you hit deploy and then go to ENV section drop down, it still states LEAP APIs

**Comment by burgalon** [comment]
Can you just ignore those env vars for now?

**Comment by D2OKAY** [comment]
> Can you just ignore those env vars for now?

Yes, and also the link from the README is different from the vecel template link. 
I can confirm that the README 'deploy' works with correct ASTRIA_API_KEY and not LEAP 

Thank you so much, trying it now =)

**Comment by kyavichev** [comment]
I just went through this using template from Vercel and it still asks for Leap stuff. You cannot fully deploy as it can't resolve keys, so you end up with a project that is not fully configured. (supabase is not configured, which took a while to sort out)

**Comment by burgalon** [comment]
@kyavichev which leap stuff exactly ? Screenshot ?

**Comment by kyavichev** [comment]
If you go to Vercel, create a new project using a template, select Headshot Generator one, it will take you to the page where you need to input LEAP keys:

<img width="721" alt="Image" src="https://github.com/user-attachments/assets/51717545-4654-40e5-9e56-aafa923c60e8" />

<img width="712" alt="Image" src="https://github.com/user-attachments/assets/5dc921d4-d4c3-45cf-bf6c-81c066d0572e" />

It's not an issue with this repo or Astria, just Vercel template, but it is confusing for newbies like me.

**Comment by MichaelBicker** [comment]
The template still asks for a Leap AI API key and makes it not deployable this way (when trying to enter real leap ai credentials)

**Comment by burgalon** [comment]
@MichaelBicker please use the deploy button from the README and not from the Vercel marketplace. We need to ask Vercel to update the template
